{{ cookiecutter.app_name }}/pyproject.toml

```diff
@@ -163,17 +163,25 @@ system_runtime_requires = [
 ]
 
 [tool.briefcase.app.{{ cookiecutter.app_name }}.linux.appimage]
+manylinux = "manylinux2014"
+
 system_requires = [
 {%- if cookiecutter.gui_framework == "Toga" %}
-    "libcairo2-dev",
-    "libgirepository1.0-dev",
-    "libgtk-3-dev",
-    "libpango1.0-dev",
-    "librsvg2-dev",
-    # Needed to support Webkit2
-    # "gir1.2-webkit2-4.0",
-    # "libwebkit2gtk-4.0-dev",
-{% endif -%}
+    # Needed to compile pycairo wheel
+    'cairo-gobject-devel',
+    # Needed to compile PyGObject wheel
+    'gobject-introspection-devel',
+    # Needed to provide GTK
+    "gtk3-devel",
+    # Needed to provide WebKit2 at runtime
+    # "webkit2gtk3",
+{%- elif cookiecutter.gui_framework == "PySide2" %}
+    "qt5-qtbase-gui",
+{%- elif cookiecutter.gui_framework == "PySide6" %}
+    "qt6-qtbase-gui",
+{%- elif cookiecutter.gui_framework == "PursuedPyBear" %}
+# ?? FIXME
+{%- endif %}
 ]
```

Comment:
nitpick: the quote character differences are making my twitch a bit :) also appears to have sneaked by in the `system_requires` lists for the `linux.system` sections

Reply:
Well now *I'm* twitching too. Thanks for that :-)